docs/source/user_guide/pipeline-components.md

```diff
@@ -192,6 +192,24 @@ Note: You must specify all property values, not only the ones that you want to m
 
 Refer to section [Configuration properties](#configuration-properties) for parameter descriptions.
 
+#### Exporting component catalogs
+
+To export component catalogs:
+
+```bash
+elyra-metadata export component-catalogs \
+	--schema-name="url-catalog" \
+	--directory="/tmp/foo"
+```
+
+The above example will export all component catalogs in the "url-catalog" schema to the "/tmp/foo/component-catalogs" directory.
+
+Note that you must specify the `--directory` option. If the `--schema-name` option is not specified, component catalogs across all schemas will be exported. 
```

Comment:
```suggestion
Note that you must specify the `--directory` option. If the `--schema_name` option is not specified, component catalogs across all schemas will be exported. 
```

Reply:
Removed all mentions of `--schema-name` parameter from the documentation as discussed via chat.